package.json

```diff
@@ -105,7 +105,7 @@
     "react-router": "^4.2.0",
     "react-router-dom": "^4.2.2",
     "react-router-redux": "^5.0.0-alpha.6",
-    "react-transition-group": "^2.2.1",
+    "react-transition-group": "^1.2.0",
```

Comment:
Any reason why to use an older version of this package?

Reply:
@mitjade Ah yes I had to downgrade this because 2.0+ conflicts with some of reactstrap. I found it when I added an `Alert` to the form. See: https://github.com/reactstrap/reactstrap/issues/498